packages/lib/src/__tests__/test-lib/mock-logger.ts

```diff
@@ -0,0 +1,36 @@
+import { Logger } from '@smartthings/core-sdk'
+
+
+export const traceMock = jest.fn()
+export const debugMock = jest.fn()
+export const infoMock = jest.fn()
+export const warnMock = jest.fn()
+export const errorMock = jest.fn()
+export const fatalMock = jest.fn()
+
+export const isTraceEnabledMock = jest.fn()
+export const isDebugEnabledMock = jest.fn()
+export const isInfoEnabledMock = jest.fn()
+export const isWarnEnabledMock = jest.fn()
+export const isErrorEnabledMock = jest.fn()
+export const isFatalEnabledMock = jest.fn()
+
+export function buildMockLogger(level = 'debug'): Logger {
+	return {
+		level,
+
+		trace: traceMock,
+		debug: debugMock,
+		info: infoMock,
+		warn: warnMock,
+		error: errorMock,
+		fatal: fatalMock,
+
+		isTraceEnabled: isTraceEnabledMock,
+		isDebugEnabled: isDebugEnabledMock,
+		isInfoEnabled: isInfoEnabledMock,
+		isWarnEnabled: isWarnEnabledMock,
+		isErrorEnabled: isErrorEnabledMock,
+		isFatalEnabled: isFatalEnabledMock,
+	}
+}
```

Comment:
An alternative method of implementing this would be using [jest manual mocks](https://jestjs.io/docs/en/manual-mocks#mocking-user-modules). Then tests could just call `jest.mock('logger')` to use it.

Reply:
I used a manual mock for the `../logger` file but not for all of `@smartthings/core-sdk` so this file is still here but now  used in the manual mock for the local logger file.